packages/bits/src/styles/mixins/focus.less

```diff
@@ -1,6 +1,5 @@
 .focus-outline-base() {
     outline: none;
-    z-index: @zindex-active;
```

Comment:
@Vita-Karas, I would leave this here since it's been there for a couple of years. It seems too risky to remove it. It might be good to write a ticket to consider removing it on a major Nova version bump, though.

Reply:
Created ticket: [NUI-6103](https://jira.solarwinds.com/browse/NUI-6103)